libraries/libfc/include/fc/io/random_access_file.hpp

```diff
@@ -419,12 +433,14 @@ class random_access_file {
    void pack_to(const T& v, const ssize_t offset) {
       write_datastream ds(ctx, offset);
       fc::raw::pack(ds, v);
+      ds.flush();
```

Comment:
Could the `ctx->write_to()` on each `pack_to()` call hurt performance?

Reply:
Not in this case: the buffer is/was flushed upon destruction of `write_datastream` so the buffer was already being flushed at the end of `pack_to()`.